tests/playbooks/tests_infiniband.yml

```diff
@@ -2,6 +2,9 @@
 ---
 - name: Play for testing infiniband device
   hosts: all
+  vars:
+    interface: "{{ lookup('env', 'INFINIBAND_INTERFACE_NAME') |
+      default('INFINIBAND_INTERFACE_NAME', 'ib0') }}"
```

Comment:
```suggestion
      default(lookup('env', 'INFINIBAND_INTERFACE_NAME'), 'ib0') }}"
```

Reply:
if you don't want to have to do the lookup twice - then define another var:
```yaml
vars:
  env_interface: "{{ lookup('env', 'INFINIBAND_INTERFACE_NAME') }}"
  interface: "{{ env_interface | ternary(env_interface, 'ib0') }}"
```

Reply:
I actually get a hint from here, https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_filters.html#providing-default-values, and I tested it successfully locally when the env var `INFINIBAND_INTERFACE_NAME` is defined and not defined.